packaging/ansible-runner-service-project/project/roles/ovirt-ova-pack/tasks/main.yml

```diff
@@ -1,16 +1,34 @@
 ---
+- name: Create temporary directory
+  tempfile:
+    state: directory
+    suffix: ova
+  register: ova_temp
+
+- name: Copy pack_ova.py to temp directory
+  copy:
+    src: pack_ova.py
+    dest: "{{ ova_temp.path }}/pack_ova.py.py"
```

Comment:
Watch out for the double `.py` please change it to `pack_ova.py`

Reply:
Ah good catch, thanks